test/org/github/jamm/GuessTest.java

```diff
@@ -46,167 +38,154 @@
 @RunWith(Parameterized.class)
 public class GuessTest {
 
-    @SuppressWarnings("deprecation")
     @Parameterized.Parameters
     public static Collection<MemoryMeter.Guess> guesses() {
+
```

Comment:
Not sure I understand your problem with the command line.

Reply:
```
But on command line everything succeeds. It just shows only instrumentation even if the test is parameterized.
Shall we make it print info about each strategy when it uses it in these tests?
```
So the tests are parameterized to run with every strategy but this is the info I see:
```
[INFO] Running org.github.jamm.GuessTest
UseEmptySlotsInSupers: null
WARNING: Jamm is starting with the UseEmptySlotsInSupers JVM option disabled. The memory layout created when this option is enabled cannot always be reproduced accuratly by the SPEC or UNSAFE strategies. By consequence the measured sizes when these strategies are used might be off in some cases.
Jamm starting with: java.version='1.8.0_231', java.vendor='Oracle Corporation', instrumentation=true, unsafe=true, Memory Layout: [objectHeaderSize=12 , arrayHeaderSize=16, objectAlignement=8, referenceSize=4]
[INFO] Tests run: 6, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 16.695 s - in org.github.jamm.GuessTest
```
I was thinking maybe it can show info per strategy run but now on a second thought I think it is fine. We just say what is available. My only concern is when we use the different flags that I always see the same Memory Layout?
`Memory Layout: [objectHeaderSize=12 , arrayHeaderSize=16, objectAlignement=8, referenceSize=4]`